Source/Fuse.Elements/Caching/ElementBatch.uno

```diff
@@ -140,12 +140,28 @@ namespace Fuse.Elements
 				elm.AbsoluteZoom)), CachingRectPadding);
 		}
 
+		Box GetRenderBounds(Element elm)
+		{
+			var t = elm.InternLocalTransformInternal;
+			return VisualBounds.BoxTransform((Box)elm.LocalRenderBounds, t);
+		}
+
 		VisualBounds CalcRenderBounds()
 		{
-			var rect = VisualBounds.Empty;
-			for (int i = 0; i < _elements.Count; i++)
-				rect = rect.Merge(_elements[i]._elm.CalcRenderBoundsInParentSpace());
-			return rect;
+			if (_elements.Count == 0) return VisualBounds.Empty;
+
+			var box = GetRenderBounds(_elements[0]._elm);
+			for (int i = 1; i < _elements.Count; i++)
+			{
+				var b = GetRenderBounds(_elements[i]._elm);
```

Comment:
This doesn't seem to account for a child returning Infinite or Empty for the render bounds.